caffe2/python/trt/test_trt.py

```diff
@@ -131,46 +131,48 @@ def _test_onnx_importer(self, model_name, data_input_index,
         ws = Workspace()
         with core.DeviceScope(device_option):
             ws.FeedBlob(op_inputs[data_input_index], data)
+            if opset_version >= 5:
+                ws.FeedBlob("data_0", data)
```

Comment:
Looks like we are hard coded the input name. If so, please leave some comments to explain it.

Reply:
Done. Thanks!